scripts/Testing/Testcases/HOMME_build.csh

```diff
@@ -0,0 +1,35 @@
+#!/bin/csh -f
+
+./Tools/check_lockedfiles || exit -1
+
+setenv COMPILER `./xmlquery COMPILER -value`
+source env_mach_specific || exit -1
+
```

Comment:
Yes, I realize this is a hack. I was tasked with getting Homme testing into our nightly test suites and I thought the easiest way to do this was to make a HOMME testcase. I did not want to be in the situation where some of our tests run through create_test via test cases and others are unit-tests.